account_product_fiscal_classification/tests/tests.py

```diff
@@ -138,3 +144,12 @@ def test_06_unlink_fiscal_classification(self):
             [self.classification_1_id])
         with self.assertRaises(ValidationError):
             classification.unlink()
+
+    def test_07_classification_generate_from_template(self):
```

Comment:
so in case somebody is looking at that, I wondered if eventually the Odoo TransactionCase test would run at install and not later at update, so I tried this change:
https://github.com/OCA/account-fiscal-rule/commit/1cbc107b75f6f79cb2e72e28928e11cfd2a986e4
Sadly that didn't change anything according to the coveralls results